DVCLive: Deprecate `live` section of `dvc.yaml`. (#3411)
iterative/dvc.org: content/docs/dvclive/dvclive-with-dvc.md

  `dvc metrics` and `dvc plots`. Those same outputs can be visualized in
  [Iterative Studio](#iterative-studio).

> - DVCLive is also capable of generating [checkpoint](#checkpoints) signal files

Reviewer: "several useful ways" now down to 2. Should we say "a couple useful ways" or expand the list?

  used by DVC <abbr>experiments<abbr>.

## Setup

We will refer to a training script (`train.py`) already using `dvclive`:

<admon type="tip">

If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you

> can jump to its corresponding page to find an example usage.

Reviewer: Link?
```suggestion
can jump to its [corresponding page](/doc/dvclive/ml-frameworks) to find
example usage.
```

Author: That link is already present in the previous line. Do you think it's worth repeating?

Reviewer: I guess I didn't notice the link. It's OK then!

`dvc init` first):

```dvc
$ dvc stage add \
--name train \
--deps train.py \
--metrics-no-cache training_metrics.json \
--plots-no-cache training_metrics/scalars \
python train.py
```

> <admon type="exclamation">

Reviewer: No such type that I know of. Only info, tip, warn.

Author: It is one of the supported types:

https://github.com/iterative/dvc.org/blob/788b549dab46a2fddddf86180660a3c4348ba936/plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Admonition/index.tsx#L5-L13

Reviewer: I see! They only mentioned 3 in the description of https://github.com/iterative/cml.dev/pull/210. Maybe people have been adding more emojis but I'm still not sure that each type will have a different style (background color, etc.)

Reviewer: Ya. It looks like it uses the same style as warn.

<img width="779" alt="image" src="https://user-images.githubusercontent.com/1477535/166838869-aa5e6285-b2c9-490c-bafb-f70d3f2d560c.png">
> See https://dvcorg-deprecatelive.gtsb.io/doc/dvclive/dvclive-with-dvc

Reviewer: We only use info, warn, tip in the codebase so far. I think we should stick to those if possible. I'll check with the web team on why there's so many types.

```suggestion
<admon type="warn">
```

Reviewer 2: > It is one of the supported types:

Those aren't actually types. There are only three type options:

1. info (creates purple and default emoji is ℹ️)
2. tip (creates blue and default emoji is 💡)
3. warn (creates orange and default emoji is ⚠️)

Those are `icon` options! Which you can use in combination with `type`! Since you want an exclamation `icon`: 
```suggestion
<admon type="warn" icon="exclamation">
```

Reviewer: Thanks for clarifying @julieg18 !

So up to you @daavoo but I vote for plain `warn`.

> `training_metrics.json` can be used by `dvc metrics` and visualized with

Reviewer: CE (matches https://github.com/iterative/dvc.org/pull/3411#pullrequestreview-958511443 above)
```suggestion
in `training_metrics.json` can be used by `dvc metrics` and visualized with
```
> May be needed elsewhere? This is pretty minor tho 💅🏼 up to you to accept it or not

`dvc exp show`/`dvc exp diff`.

The [metrics history](/doc/dvclive/api-reference/live/log#step-updates)
`training_metrics/scalars` can be visualized with `dvc plots`.

> The [HTML report](/doc/dvclive/api-reference/live/make_report##description)
> `training_metrics/report.html` will contain all the logged data and will be

Reviewer: CE
```suggestion
The [HTML report](/doc/dvclive/api-reference/live/make_report#description)
in `training_metrics/report.html` will contain all the logged data and will be
```
